### PR TITLE
Seperate deployment google azure

### PR DESCRIPTION
**Pull request by gabypascha**

### CODE REVIEW

File: DOc/docs/source/Options/Deployment/docker_compose_deployment.rst [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+No additional action is required
+
+Deployment on VM with public domain name
+===========================================
+Connect to the VM using as destination its public IP
+
+Deployment on VM without public domain name
+==============================================
+
+In this deployment situation, a VM is used further referred to as Host and a client, whcih can be directly accessed by the user.
```
Comment (wombach) [inline DOc/docs/source/Options/Deployment/docker_compose_deployment.rst]: ... a VM is further... no "used"

File: DOc/docs/source/Options/Deployment/docker_compose_deployment.rst [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+.. code:: bash
+
+	8087 -> 127.0.0.1:8087
+
+- Extend hosts file on the client with the following line (admin right required)
+
+.. code:: bash
+
+	127.0.0.1       localhost localhost4 $EXTERNAL\_IP
```
Comment (wombach) [inline DOc/docs/source/Options/Deployment/docker_compose_deployment.rst]: I thought it is the external Hostname and not the IP address... at least this is what I did.

File: DOc/docs/source/Options/Deployment/docker_compose_deployment.rst [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+.. code:: bash
+
+	export EXTERNAL\_IP=$(ifconfig eth0 | grep 'inet' | cut \-d: \-f2 | sed \-e 's/.\*inet \\([^ ]\*\\).\*/\\1/')
+
+
+- If deployment is on a VM:
+
+.. code:: bash
+
+	export EXTERNAL\_IP={hostname of VM}
```
Comment (wombach) [inline DOc/docs/source/Options/Deployment/docker_compose_deployment.rst]: it is confusing to use a variable external_IP if you use a hostname and NOT an IP address.

File: DOc/docs/source/Options/Deployment/docker_compose_deployment.rst [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+- Aurelius/Apache Atlas Admin User:  
+	Username: atlas  
+
+	Password: 1234
+
+Spin up docker-compose environment:
+=====================================
+ 
+To start up the system, execute the following command on the host.
```
Comment (wombach) [inline DOc/docs/source/Options/Deployment/docker_compose_deployment.rst:143]: calling the docker-compose.yml file included in this distribution.

File: DOc/docs/source/Options/Deployment/docker_compose_deployment.rst [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+Starting up the system may take several minutes. 
+
+This is how the system looks in operational state:
+
+.. image:: ../imgs/docker_compose_ps.png
+
+When the Apache Atlas container state changes from starting to healthy, then the system is ready.
+
+
+You are able now to access Aurelius Atlas at thw client wiht the URL `http://$EXTERNAL_IP:8087/`
```
Comment (wombach) [inline DOc/docs/source/Options/Deployment/docker_compose_deployment.rst]: type: the

**Review by wombach — APPROVED** [review]
a few small changes... please correct and then merge

File: DOc/docs/source/Options/Deployment/docker_compose_deployment.rst (hunk 5 above)
Comment (mthomas-aurelius) [reply DOc/docs/source/Options/Deployment/docker_compose_deployment.rst]: done

File: DOc/docs/source/Options/Deployment/docker_compose_deployment.rst (hunk 4 above)
Comment (mthomas-aurelius) [reply DOc/docs/source/Options/Deployment/docker_compose_deployment.rst:143]: done

File: DOc/docs/source/Options/Deployment/docker_compose_deployment.rst (hunk 2 above)
Comment (mthomas-aurelius) [reply DOc/docs/source/Options/Deployment/docker_compose_deployment.rst]: we discussed this point, depends on the deployment scenario

File: DOc/docs/source/Options/Deployment/docker_compose_deployment.rst (hunk 1 above)
Comment (mthomas-aurelius) [reply DOc/docs/source/Options/Deployment/docker_compose_deployment.rst]: removed the whole sentence

File: DOc/docs/source/Options/Deployment/docker_compose_deployment.rst (hunk 3 above)
Comment (mthomas-aurelius) [reply DOc/docs/source/Options/Deployment/docker_compose_deployment.rst]: I agree, changing EXTERNAL_IP with EXTERNAL_HOST

**Pull request merged**